app/services/katello/registration_manager.rb

```diff
@@ -87,6 +87,10 @@ def validate_hosts(hosts, organization, host_name, host_uuid, host_uuid_overridd
           host = hosts.first
 
           if host.name == host_name
+            if !host.build && Setting[:host_re_register_build_only]
+              registration_error("Host with name %{host_name} is currently registered but not in build mode (host_re_register_build_only==True).  You can unregister the host manually or put it into build mode to continue.", host_name: host_name)
```

Comment:
Looks like there are two spaces in between the sentences. Also recommend removing "you" ie `Unregister the host manually or <snip>`

Reply:
Fixed :)